crates/committer/src/storage/storage_trait.rs

```diff
@@ -29,32 +29,38 @@ pub(crate) trait Storage {
 }
 
 pub(crate) enum StoragePrefix {
-    PatriciaNode,
+    InnerNode,
+    StorageLeaf,
+    StateTreeLeaf,
+    CompiledClassLeaf,
 }
 
+/// Describes a storage prefix as used in Aerospike DB.
 impl StoragePrefix {
     pub(crate) fn to_bytes(&self) -> &[u8] {
         match self {
-            Self::PatriciaNode => "patricia_node:".as_bytes(),
+            Self::InnerNode => b"patricia_node",
+            Self::StorageLeaf => b"starknet_storage_leaf",
+            Self::StateTreeLeaf => b"contract_state",
+            Self::CompiledClassLeaf => b"contract_class_leaf",
         }
     }
 }
 
 impl StorageKey {
     pub(crate) fn with_prefix(&self, prefix: StoragePrefix) -> Self {
```

Comment:
In my opinion, this logic is confusing and will run slower than using create_db_key